src/components/data-table/column-sorting/column-sort.service.ts

```diff
@@ -0,0 +1,43 @@
+import {Injectable} from '@angular/core';
+import {Subject} from 'rxjs/Subject';
+import {
+  SortDirection,
+  IColumnSortingModel
+} from './column-sort';
+
+/**
+ * Service handling column sort change events as an observable stream.
+ */
+@Injectable()
+export class MdDataColumnSortingService {
+  public sortingColumn$: Subject<IColumnSortingModel> = new Subject();
```

Comment:
This should really be a `BehaviorSubject` so we can assign a default for the view more easily.